website/docs/docs/cloud/about-cloud-develop-defer.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Using defer in dbt Cloud
+id: about-cloud-develop-defer
+description: "Learn how to leverage defer to prod when developing with dbt Cloud."
+sidebar_label: "Using defer in dbt Cloud"
+pagination_next: "docs/cloud/cloud-cli-installation"
+---
+
+
+[Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build these models' upstream parents. This is powered by having a production manifest to compare to, and dbt will resolve the `{{ ref() }}` function with upstream production artifacts.
+
+Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows, dramatically reducing development time and warehouse spend by preventing unecessary model builds. 
+
+## Required setup
+
+Set your production environment using the [production environment checkbox](/docs/deploy/deploy-environments#set-as-production-environment-beta) in the environment settings page. This can be set for one deployment environment per dbt Cloud project. When using defer, the artifacts from the latest successful, non-CI job execution in the production environment will be compared to.
```

Comment:
```suggestion
- You must select the **[Production environment](/docs/deploy/deploy-environments#set-as-production-environment-beta)** checkbox in the **Environment Settings** page. 
  - This can be set for one deployment environment per dbt Cloud project. 
  
When using 'defer', it compares artifacts from the most recent successful production job, excluding CI jobs.
```